frontend/src/components/SepForm.js

```diff
@@ -16,19 +27,329 @@ const useStyles = makeStyles({
+                    id="last_name"
+                    name="last_name"
+                    onChange={handleChange}
+                    value={values.last_name}
+                  />
+                </FormControl>
+              </Grid>
+              <Grid item xs={12}>
+                <FormControl
+                  error={errors.date_of_birth && touched.date_of_birth}
+                  disabled={participantId ? true : false}
+                >
+                  <InputLabel shrink htmlFor="date_of_birth">
+                    Date of Birth
+                  </InputLabel>
+                  <PrevPointInput
+                    id="date_of_birth"
+                    name="date_of_birth"
+                    type="date"
+                    onChange={handleChange}
+                    value={values.date_of_birth}
+                  />
+                </FormControl>
+              </Grid>
+              <Grid item xs={12}>
+                <FormControl
+                  error={errors.maiden_name && touched.maiden_name}
+                  disabled={participantId ? true : false}
+                >
+                  <InputLabel htmlFor="maiden_name">
+                    Mother&apos;s Maiden Name
+                  </InputLabel>
+                  <PrevPointInput
+                    id="maiden_name"
+                    name="maiden_name"
+                    onChange={handleChange}
+                    value={values.maiden_name}
+                  />
+                </FormControl>
+              </Grid>
+              <Grid item xs={12}>
+                <PrevPointButton
+                  type="submit"
+                  disabled={(participantId ? true : false) || isSubmitting}
+                >
+                  Search
+                </PrevPointButton>
+                {(errors.sep_id ||
+                  errors.last_name ||
+                  errors.date_of_birth ||
+                  errors.maiden_name) && (
+                  <PrevPointCopy className={classes.errorMessage}>
+                    {errors.sep_id ||
+                      errors.last_name ||
+                      errors.date_of_birth ||
+                      errors.maiden_name}
+                  </PrevPointCopy>
+                )}
+              </Grid>
+            </Grid>
+          </Form>
+        )}
+      </Formik>
+      <Formik
+        className={classes.form}
+        validateOnChange={false}
+        validateOnBlur={false}
+        initialValues={{
+          needles_in: "",
+          needles_out: "",
+          visit_date: "",
+        }}
+        validationSchema={SEPNeedleSchema}
+        onSubmit={async (values, { setSubmitting }) => {
+          await participantStore.createSEP({
+            needles_in: values.needles_in,
+            needles_out: values.needles_out,
+            site: participantStore.currentSite,
+            participant: participantId,
+            urgency: 1,
+            program: 10,
+          })
```

Comment:
Here is where the values for CreateVisits and CreateSEP are passed. I've left urgency and program hardcoded for now because I am getting an error on the CreateVisits call.
{"participant":["Incorrect type. Expected pk value, received str."]}

Reply:
i left a comment above addressing this